src/XRP/issued-currency-client.ts

```diff
@@ -453,6 +453,32 @@ export default class IssuedCurrencyClient {
     )
   }
 
+  /**
+   * Freezes the trust line between this account (issuing account) and another account.
+   * Note that the trust line's limit is set to 0.
+   *
+   * @see https://xrpl.org/freezes.html#enabling-or-disabling-individual-freeze
+   *
+   * @param accountToFreeze The X-Address of the account which to to freeze a trust line.
```

Comment:
I think we actually need a better name for this parameter, since technically you can actually freeze a whole account (that's what the global freeze is).  Don't want to create confusion there... we could call it `counterpartyAccount` although we've never used that term anywhere else... we could call it `accountOnTrustline` or `trustlineAccount` or `trustlineCounterparty` or....  And actually, maybe we could rename `currencyName` below to `trustlineCurrencyName` and then use `trustlineAccount` to make a little more clear and consistent that these data are what specify the trustline that we're freezing.

Reply:
gotcha, that's a great point